### DISCUSSION TITLE
добавте пожалуйста

### DISCUSSION THREAD

**Discussion by xgyra4ku** [stoatchat/discussions#798] [Feature Suggestions]
добавте пожалуйста демонстрацию экрана и камеруу пожалуйса хотелось бы чтобы еще когда говоришь был зеленый кружок как в дискорде и отображалось кто в голосовом канале

**Comment by Mixail89** [comment]
надо надо)

**Comment by INAAAXXX** [comment]
в принципе было бы неплохо добавить вход в войс чат через одно нажатие, а не сначала зайти в канал чата, а потом уже нажать кнопку подключения
